manager-bundle/src/Resources/web/app.php

```diff
@@ -35,6 +35,11 @@
 
 // Handle the request
 $request = Request::createFromGlobals();
+
+if ($request->isSecure()) {
+    ini_set('session.cookie_secure', 'On'); // see #1760
```

Comment:
isn't it common to check `method_exists('ini_set')` and mute the call with `@ini_set`

Reply:
You are right. Fixed in 0b47eae1831a449caf0b4cc20f5918df80440202.

Reply:
`function_exists` isn't necessary? https://github.com/composer/composer/blob/master/bin/composer#L26

Reply:
I don't think so. We are not using it in line 17, either:

https://github.com/contao/contao/blob/30109229844aa9f94bf679d4cc0c8524f94535bc/manager-bundle/src/Resources/web/app.php#L16-L17

Reply:
May be the shut up operator also suppresses any errors if `ini_set` isn't available/defined in disabled functions? If so we could add it for `ini_set('session.cookie_secure', 'On');` as well.